src/components/Chart/ChartSquare.js

```diff
@@ -28,7 +28,7 @@ export const ChartSquare = ({ data, type }) => {
   };
 
   const style = {
-    gridColumnStart: episodeNumber,
+    gridColumnStart: type === 'yearLabels' ? episodeNumber : null,
```

Comment:
fixes bug where year number is not directly to the left of first episode in that season